### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by jonaslimamelo** [frontendbr/forum#2410] [Perguntas e Respostas]
npm ERR! Missing script: "dev"
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\intel 05\AppData\Local\npm-cache\_logs\2023-05-19T16_31_20_209Z-debug-0.log
PS C:\Users\intel 05\Desktop\projects>

**Comment by jonaslimamelo** [comment]
alguem sabe como resolver

**Comment by chriscorrales** [comment]
Seu ```package.json``` não tem um script chamado `dev`.
Tem que ter algo assim como nesse exemplo: 
```json
{
  "name": "checkout-sun",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "react-scripts start"
  },

```
